app/routes.tsx

```diff
@@ -160,7 +160,7 @@ export const routes = createRoutesFromElements(
           loader={InventoryPage.loader}
           handle={{ crumb: 'Inventory' }}
         >
-          <Route index element={<Navigate to="sleds" replace />} loader={SledsTab.loader} />
+          <Route index element={null} loader={() => replace('sleds')} />
```

Comment:
The nulls are because you get warnings if you don't have an element to render, even if the loader returns a redirect.